Mathlib/MeasureTheory/Constructions/BorelSpace/Real.lean

```diff
@@ -465,6 +466,73 @@ _root_.measurable_of_tendsto_nnreal := NNReal.measurable_of_tendsto
 
 end NNReal
 
+namespace EReal
+
+lemma measurableEmbedding_coe : MeasurableEmbedding Real.toEReal :=
+  isOpenEmbedding_coe.measurableEmbedding
+
+instance : MeasurableAdd₂ EReal := ⟨EReal.lowerSemicontinuous_add.measurable⟩
+
+section MeasurableMul
+
+variable {α β γ : Type*} {mα : MeasurableSpace α} {mβ : MeasurableSpace β} {mγ : MeasurableSpace γ}
+
+@[fun_prop]
```

Comment:
I'm not sure this should be tagged with `fun_prop`: it's not the kind of lemma I would apply in an automatic procedure (and neither `measurable_of_real_real`) because it gives rise to more complicated expressions in a typical situation.

Reply:
I don't really mind because I guess it won't apply very often, but the reason I tagged it was that it was analogous to `Measurable.of_uncurry_left`/`Measurable.of_uncurry_right`